wishlist/app/com/commercetools/sunrise/wishlist/controllers/ClearWishlistControllerAction.java

```diff
@@ -9,6 +9,13 @@
 
 @ImplementedBy(DefaultClearWishlistControllerAction.class)
 public interface ClearWishlistControllerAction extends ControllerAction, Function<ShoppingList, CompletionStage<ShoppingList>> {
+    /**
+     * Removes all line items from the given wishlist.
+     *
+     * @param wishlist the wishlist
+     *
+     * @return the completion stage for this action
+     */
```

Comment:
Maybe it's better to have a class javadoc than having it for the method, as in this case (as in most cases) `@param` and `@return` documentation are too verbose and provide no additional info. That, or be more specific with the description:
```
    * @param wishlist the {@link shoppingList} from which to remove all line items
    *
    * @return the completion stage for the updated wishlist with no items.
```